caldav/server.go

```diff
@@ -209,8 +227,30 @@ func (b *backend) HeadGet(w http.ResponseWriter, r *http.Request) error {
 }
 
 func (b *backend) Propfind(r *http.Request, propfind *internal.Propfind, depth internal.Depth) (*internal.Multistatus, error) {
+	homeSetPath, err := b.Backend.CalendarHomeSetPath(r.Context())
+	if err != nil {
+		return nil, err
+	}
+	principalPath, err := b.Backend.CurrentUserPrincipal(r.Context())
+	if err != nil {
+		return nil, err
+	}
+
 	var resps []internal.Response
-	if r.URL.Path == "/" {
+
+	resp, err := b.propfindCommon(propfind, r.URL.Path, principalPath)
```

Comment:
Hmm. This is a bit weird. It produces multiple `<response>` elements for the same href potentially.

Reply:
Yeah, sorry, I think I refactored that once too often. The composability is a bit weird there. Are you okay with how it is now?

I think in the long run, if we add more basic WebDAV properties, it might make sense to pass around pre-filled maps. But I tried that, and it would require a map copy in some places, so it's a bit ugly also. For now, it seemed best to just follow the current pattern but add the additional property everywhere...

Reply:
> Are you okay with how it is now?

Yeah sounds fine, let's keep it simple. Though, CardDAV seems fixed but not CalDAV.

Reply:
Sorry, that got lost in rebase-land :confused: fixed now